core/openapi.yaml

```diff
@@ -196,6 +196,54 @@ paths:
                     type: array
         400:
           description: The request body is invalid
+
+  /project_signals:
+    post:
+      tags:
+        - simulation
+      summary: Compute signal updates
+      operationId: simulation
```

Comment:
```
Semantic error at paths./project_signals.post.operationId
Operations must have unique operationIds.
```

(According to editor.swagger.io)

There's other errors in the file but not related to these changes.

Reply:
Oooh. That's better than staring at the file really hard ^^'
Fixed.